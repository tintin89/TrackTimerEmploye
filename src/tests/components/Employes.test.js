import { shallow } from 'enzyme';

import Employes from '../../components/Employers/Employers'


describe('test in Employes.js',()=>{
  const wrapper = shallow(<Employes/>)

    test('Must show correctly',()=>{
     
        expect(wrapper).toMatchSnapshot();
    })
})