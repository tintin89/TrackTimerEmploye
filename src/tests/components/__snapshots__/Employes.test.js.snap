// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test in Employes.js Must show correctly 1`] = `
<div
  className="resultsEmployes"
/>
`;
